src/nav.ts

```
@@ -1,294 +1,313 @@
+    }
 }
-function openMenu()  { if (navMenu){ navMenu.classList.remove("hidden"); applyMobile(true);} }
-function closeMenu() { if (navMenu){ applyMobile(false); navMenu.classList.add("hidden");} }
 
-BURGER?.addEventListener("click", () =>
-  (navMenu && navMenu.classList.contains("hidden") ? openMenu() : closeMenu())
-);
-navMenu?.querySelectorAll("button").forEach(btn =>
-  btn.addEventListener("click", () => innerWidth < MOBILE_BP && closeMenu())
+BURGER?.addEventListener('click', () =>
+    navMenu && navMenu.classList.contains('hidden') ? openMenu() : closeMenu()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)